Michael Wang JSON

```diff
@@ -0,0 +1 @@
+{name: "Michael Wang"}
```

Comment:
This files should:
* be placed under `data` folder like other data files
* named something like `michaelwang.json`

Take a look at others' examples in the data folder!